.pre-commit-config.yaml

```diff
@@ -34,3 +34,10 @@ repos:
     hooks:
       - id: giticket
         args: ['--regex', '[A-Z]+-\d+[0-9]', '--format', '{ticket}: {commit_msg}', '--mode', 'regex_match']
+
+  - repo: local
+    hooks:
+      - id: custom
```

Comment:
I would use a more descriptive id is possible

Reply:
Will change